### PR TITLE
add hackathon line

### PR DESCRIPTION
**Pull request by emilyJLin95**

### CODE REVIEW

**Review by cedrabner — COMMENTED** [review]
i think phrasing is a bit off. maybe change to "the easiest way to get test tokens is by sending..."

**Comment by emilyJLin95** [comment]
@cedrabner I agree with your statement, but this is very specific for hackathon developers. Do you have alternative suggestion so that that cohort of people know where to go?

File: docs/use-mainnet/fund.md [hunk 1]
```diff
@@ -6,7 +6,9 @@ sidebar_position: 2
 
 ## Getting tokens on Linea
 
-There are several ways to get tokens on Linea:
+For hackathon developers, the easiest way to get test token is sending a message via https://xmtp.chat to an ENS name provided in the relevant hackathon prizes page.
```
[the remark this answers is not in the document]
Comment (emilyJLin95) [reply docs/use-mainnet/fund.md]: okay addressed!

**Review by cedrabner — APPROVED** [review]
lgtm

**Pull request merged**